SUPPORTED_OPS.rst

```diff
@@ -0,0 +1,72 @@
+Supported Operations
+####################
+
+.. table::
+
+    +------------------+-----------------------------------------+--------------+-----------------------+
+    | **Visibility**   | **Operation**                           | **Protocol** | **Number of Parties** |
+    +==================+=========================================+==============+=======================+
+    | Public & Private | Addition                                | - Falcon     |                    2+ |
+    |                  |                                         | - FSS        |                       |
+    |                  |                                         | - SPDZ       |                       |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Subtraction                             | - Falcon     |                    2+ |
+    |                  |                                         | - FSS        |                       |
+    |                  |                                         | - SPDZ       |                       |
+    |                  +-----------------------------------------+--------------+-----------------------+
+    |                  | Mult, matmul, conv2d, conv2d _transpose | - Falcon     |                    2+ |
```

Comment:
NIT: Falcon does not support mul,matmul for arbitrary parties.

Reply:
True, I saw that needs three parties.